modules/react/collection/lib/ListBox.tsx

```diff
@@ -32,8 +32,17 @@ const useListBox = createElemPropsHook(useListModel)(model => {
 });
 
 /**
- * Basic list box that supports virtualization. `ListBox.Item` contains a very simple list item
- * without much functionality. The `ListBox` contains two `Box` elements:
+ * The `ListBox` is a basic component that offers vertical rendering of a collection in the form of
```

Comment:
I don't know what "basic component" means in relation to all of our other components.
```suggestion
 * The `ListBox` component offers vertical rendering of a collection in the form of
```
^ will need Prettier applied to format length properly.